### PR TITLE
Add to eager tests to xfail while they are fixed

### PR DESCRIPTION
**Pull request by powderluv**
Issue tracked at https://github.com/llvm/torch-mlir/issues/789

### CODE REVIEW

**Comment by silvasean** [comment]
LGTM

File: python/test/eager_mode/annotate_args.py [hunk 1]
```diff
@@ -21,7 +21,8 @@
 # CHECK: Torch Tensor (shape=(1, 3, 32, 32), dtype=torch.float32)
 # -----
 # CHECK: PASS - simple
```
Comment (silvasean) [inline python/test/eager_mode/annotate_args.py:23]: I think you have to change this line to XFAIL too

**Review by silvasean — APPROVED** [review]

File: python/test/eager_mode/annotate_args.py (hunk 1 above)
Comment (makslevental) [reply python/test/eager_mode/annotate_args.py:23]: Yes that's correct (and a weakness of this testing harness that I should think about).

**Comment by powderluv** [comment]
taken over in https://github.com/llvm/torch-mlir/pull/793

**Pull request closed without merging**